src/uwtools/templater.py

```diff
@@ -70,9 +82,15 @@ def set_template(argv):
     else:
         cfg = os.environ
 
+    if user_args.config_items:
+        user_settings = dict_from_config_args(user_args.config_items)
+        cfg.update(user_settings)
 
     # instantiate Jinja2 environment and template
-    env = Environment(loader=FileSystemLoader(user_args.input_template))
+    env = Environment(
+        loader=FileSystemLoader(user_args.input_template),
+        undefined=StrictUndefined,
```

Comment:
Here, we want to make sure to fail if the values needed by the template were not provided.